samples/subsys/usb/cdc_acm_echo/src/main.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Sample echo app for CDC ACM class
+ *
+ * Sample app for USB CDC ACM class driver. The received data is echoed back
+ * to the serial port.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <device.h>
+#include <uart.h>
+#include <zephyr.h>
+#include <ring_buffer.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(cdc_acm_echo, LOG_LEVEL_DBG);
+
+#define RING_BUF_SIZE 1024
```

Comment:
I suggest to make it default smaller and configurable over samples/subsys/usb/cdc_acm/Kconfig.